### ISSUE TITLE
examples page for loaders.gl

### ISSUE THREAD

**Issue by jckr** [uber-web/ocular#196]
If an example for loaders.gl is accessed through the navigation tree it looks fine.
if it is accessed directly through its url it looks garbled:

![Screen Shot 2019-05-26 at 7 27 55 PM](https://user-images.githubusercontent.com/9727630/58391912-801e7580-7fec-11e9-99db-fe392107906b.png)

**Comment by jckr** [comment]
notes: the generated markdowns for the page generated from navigation tree ("NAV") and from its own url ("URL") are very different. 

NAV:

```
<div>
  <div>
    <div>
      <div class="markdown-body">
        <div>
          <header />
        </div>
        <div>
           <div class="toc" />
        </div>
        <div>
          <div>
             <div>
                <canvas />
                <div class="options-panel" />
              </div>
           </div>
        </div>
     </div>
  </div>
</div>
```

URL: 

```
<div>
  <div>
    <div class="options-panel">
      <div>
        <div>
          <div>
            <div>
              <header />
            </div>
          </div>
          <div>
            <div class="toc />
          </div>
          <div>
            <div>
              <div>
                <canvas />
                <div class="options-panel" />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
```

**Comment by jckr** [comment]
as stated above the problem only occurs on the built site, not on the development site.

**Comment by jckr** [comment]
Note that this problem also occurs with documentation pages, although in a less visible way. 
This is a page from its URL
![image](https://user-images.githubusercontent.com/9727630/58393076-d510ba80-7ff1-11e9-8de6-e89af2a90949.png)

This is the same page from the navigation tree
![image](https://user-images.githubusercontent.com/9727630/58393106-f70a3d00-7ff1-11e9-8806-8f9f5f6dc356.png)

**Comment by jckr** [comment]
I was looking what in the ocular-gatsby code was responsible for generating:
```
<div class="control-panel__Container-sc-14q5t5u-0 dkkqTi">
```
the generated dkkqTi class name appearing twice in the hierarchy of the examples in the URL case.
given that ocular-gatsby doesn't use styled components (at least my local version doesn't). 

This comes from the example itself which does rely on styled components.

**Comment by jckr** [comment]
it's possible, though not super elegant, to counter the double occurence of the class name by creating specific rules that would negate the extra class. ie #___gatsby > div > div > div {max-width: unset; ...}
given that these extra rules will appear in the 3rd div down from #___gatsby,

**Comment by jckr** [comment]
closed with #216.
